ironic-deployment/overlays/ipxe-builder-tls/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: baremetal-operator-system
+resources:
+- https://github.com/metal3-io/baremetal-operator/config/namespace?ref=release-0.8&timeout=120s
+- https://github.com/metal3-io/baremetal-operator/ironic-deployment/base?ref=release-0.8&timeout=120s
```

Comment:
I suppose we want to have ironic-tls here as well?

Reply:
fixed

Reply:
also changed the name of the folder